ruby/dalli/.envrc.sample

```diff
@@ -0,0 +1 @@
+export DD_API_KEY=<Your Datadog API key here>
```

Comment:
Added `.envrc` to allow users who use [Direnv](https://direnv.net/) to setup their API key without needing to modify code that's checked into Git.